src/DropDownMenu/DropDownMenu.js

```diff
@@ -11,30 +11,35 @@ import keycode from 'keycode';
 import Events from '../utils/events';
 import IconButton from '../IconButton';
 import propTypes from '../utils/propTypes';
+import rtl from '../utils/rtl';
 
 
 function getStyles(props, context) {
   const {disabled} = props;
   const spacing = context.muiTheme.baseTheme.spacing;
   const palette = context.muiTheme.baseTheme.palette;
   const accentColor = context.muiTheme.dropDownMenu.accentColor;
+  const normalizeDirection = rtl(context.muiTheme) || ((style) => {
```

Comment:
@ichiohta If you need that feature on the `master` branch, feel free to continue working on it. If you want to help with the `next` branch (our priority) then yes we can close it.